Core/DomainModel/Entity.cs

```diff
@@ -53,6 +76,20 @@ public Entity(Guid guid, Guid owner)
         public Guid Owner { get; private set; }
 
         /// <summary>
+        /// The list of tags the entity is tagged with
+        /// </summary>
+        private List<string> Tags;
+
+        /// <summary>
+        /// Returns the tag list as read only collection
+        /// </summary>
+        /// <returns>list of entity tags</returns>
+        public ReadOnlyCollection<string> GetTags()
```

Comment:
since the List<T>.AsReadOnly() returned a System.Collections.ObjectModel.ReadOnlyCollection<T> it could not be used in the get{} method, so the tag list is now private and uses this getter method to return a read only version of the list. the getter returns a FiVES.ReadOnlyCollection instead of the ObjectModel.ReadOnlyCollection to be consistent with the FiVES implementation.